controllers/actions.github.com/ephemeralrunner_controller.go

```diff
@@ -698,6 +698,31 @@ func (r *EphemeralRunnerReconciler) actionsClientFor(ctx context.Context, runner
 		opts = append(opts, actions.WithProxy(proxyFunc))
 	}
 
+	tlsConfig := runner.Spec.GitHubServerTLS
```

Comment:
Nit: extract it similar to `func (r *AutoscalingRunnerSetReconciler) actionsClientOptionsFor(...)`